src/util/logging.cpp

```diff
@@ -383,6 +400,14 @@ void Logging::initialize(
             "*.debug=true\n"
             "qt.*.debug=false");
 #endif
+
+    if (CmdlineArgs::Instance().getControllerDebug()) {
+        setLogLevel(LogLevel::Debug);
+        // This looks weird, but it's the proposed workaround for this 6 year
+        // old bug: https://bugreports.qt.io/browse/QTBUG-49704
```

Comment:
```suggestion
        // Move the the old filter to oldCategoryFilter first, because it is
        // used in controllerDebugCategoryFilter(). QT 5 does not have
        // a function to just copy the old filter. This is the proposed
        // workaround in: https://bugreports.qt.io/browse/QTBUG-49704
```

Reply:
https://github.com/mixxxdj/mixxx/pull/4523/commits/81e5fe313ac2f1b50e24f8e1f120e2cf1a5fd0de